src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -522,6 +522,11 @@ nav.sub {
 	position: relative;
 }
 
+.docblock > * {
+	max-width: 100%;
```

Comment:
I think this is actually unnecessary. According to the MDN, [`overflow: auto` should introduce new block formatting context by itself](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow). So if the element this applies to is a block element, then `max-width: 100%` is not needed. If the element this applies to is an inline element, then `max-width` is not useful because [it does not apply to the non-replaced inline elements](https://developer.mozilla.org/en-US/docs/Web/CSS/max-width).

So I think we just need
```css
.docblock > * {
	overflow-x: auto;
}
```

Or maybe
```css
.docblock > * {
	display: block;
	overflow-x: auto;
}
```
if we want to be absolutely sure, but this is likely not necessary since inline elements should be wrapped with `<p></p>` already.

Reply:
I did it for images actually. :)

Reply:
Wouldn't images be wrapped within `<p></p>`?

Reply:
Not always, for example:

```rust
//! <img src="whatever.png">
//!
//! ![alt text](https://github.com/adam-p/markdown-here/raw/master/src/common/images/icon48.png "Logo Title Text 1")
```

First one isn't but the second one is. The beauty of markdown. :D